### PR TITLE
add .travis.yml

### PR DESCRIPTION
**Pull request by 777777miSSU7777777**
Travis CI config without docker image.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+go: 
+    - 1.12.x
+
+sudo: true
+
+before_install: 
+    - go get -u golang.org/x/lint/golint
+    - go get github.com/securego/gosec/cmd/gosec/...
+    - go get golang.org/x/tools/cmd/goimports
+    - go get github.com/stretchr/testify
```
Comment (amaskalenka) [inline .travis.yml]: Why do you need `testify` here?

Comment (777777miSSU7777777) [reply .travis.yml]: > Why do you need `testify` here?
Why not? I dont need it?

Comment (amaskalenka) [reply .travis.yml]: How are you using it in travis' job?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+script:
+    - go fmt ./...
```
Comment (amaskalenka) [inline .travis.yml]: `go fmt` should print diff

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    - golint ./...
+    - gosec ./...
+    - goimports ./
```
Comment (amaskalenka) [inline .travis.yml]: `goimports` should print diff

**Comment by amaskalenka** [comment]
When either `go fmt` or `goimports` prints diff if any, the CI job must fail.

**Comment by amaskalenka** [comment]
I still haven't caught the difference between `PR` and `PUSH` checks. Can you explain it please?

**Pull request merged**